.circleci/main.yml

```diff
@@ -1513,6 +1514,73 @@ jobs:
           command: |
             poetry version << parameters.version >>
             poetry publish --build --username "$PYPI_USERNAME" --password "$PYPI_PASSWORD_PACHYDERM_SDK"
+  label-studio-ci-testing-docker-build:
+    executor: python
+    steps:
+      - setup_remote_docker:
+          version: 20.10.18
+          docker_layer_caching: true
+      - run:
+          name: docker login
+          command: echo "$DOCKER_PWD" | docker login --username pachydermbuildbot --password-stdin
+      - run:
+          name: Clone Label Studio repo
+          command: git clone https://github.com/pachyderm/label-studio.git label-studio
+      - run:
+          name: Build Docker image
+          command: |
+            cd label-studio
+            DOCKER_BUILDKIT=1 docker build -t pachyderm/label-studio-ci-testing:$CIRCLE_SHA1 .
+      - run:
+          name: Docker push
+          command: docker push pachyderm/label-studio-ci-testing:$CIRCLE_SHA1
+  test-label-studio:
+    resource_class: xlarge
+    machine:
+      image: << pipeline.parameters.machine_image >>
+    environment:
+      LABEL_STUDIO_USERNAME: user@domain.com
+      LABEL_STUDIO_PASSWORD: pass
+      LABEL_STUDIO_USER_TOKEN: abcdef
+    working_directory: ~/project/label-studio
+    steps:
+      - checkout:
+          path: ~/project
+      - install-minikube
+      - install-helm
+      - install-kubectl
+      - start-minikube
+      - wait-for-minikube
+      - attach_workspace:
+          at: ./
+      - run:
+          name: Install pachctl
+          command: |
+            sudo mv ./dist-pach/pachctl/pachctl_linux_amd64_v1/pachctl /usr/local/bin/pachctl
+      - run:
+          name: Install pach
+          command: helm install pachd ../etc/helm/pachyderm --set deployTarget=LOCAL,pachd.image.tag=${CIRCLE_SHA1},proxy.service.type=NodePort
+      - run:
+          name: Wait for pach # need to wait before testing kubectl as pach is not visible to kubectl yet
+          command: |
+            sleep 20
+            kubectl wait --for=condition=ready pod -l app=pachd --timeout=5m
+            pachctl version
+      - run:
+          name: Run Label Studio
+          command: |
+            docker run -d -it --net="host" pachyderm/label-studio-ci-testing:$CIRCLE_SHA1 label-studio -b \
+            --username $LABEL_STUDIO_USERNAME \
+            --password $LABEL_STUDIO_PASSWORD \
+            --user-token $LABEL_STUDIO_USER_TOKEN
+
+            sleep 20  # Wait for Label Studio server to start
```

Comment:
Is there a label-studio endpoint that we can poll to make this more robust/efficient?